batchflow/pipeline_executor.py

```diff
@@ -0,0 +1,321 @@
+""" Contains pipeline executor class """
+import os
+import warnings
+import traceback
+import concurrent.futures as cf
+import queue as q
+
+from .exceptions import SkipBatchException, EmptyBatchSequence, StopPipeline
+from .notifier import Notifier
+from .utils_random import make_seed_sequence
+
+
+warnings.filterwarnings("always", category=RuntimeWarning, module=__name__)
+warnings.filterwarnings("always", category=EmptyBatchSequence)
+
+END_PIPELINE = -1
```

Comment:
Maybe, `END_PIPELINE_SIGNAL`? Current naming is confusing

Reply:
Fixed